src/components/nodes/messageFlow/messageFlow.vue

```diff
@@ -45,6 +45,15 @@ export default {
     targetIsIntermediateCatchEvent(){
       return this.targetNode.definition.$type === 'bpmn:IntermediateCatchEvent';
     },
+    targetTaskTypes(){
```

Comment:
I think this is a bad name. `targetTaskType` seems to suggest this is some sort of getter while this is a query method, returns an answer to a question. I would combine this method with `targetIsIntermediateCatchEvent` since the only place that is called is in line 74 below. This could then read as follows:
`
isValidTarget(type)
return [
         'bpmn:Task', 
         'bpmn:ScriptTask', 
         'bpmn:ManualTask',
         'bpmn:CallActivity',
         'bpmn:ServiceTask',
         'bpmn:IntermediateCatchEvent',
       ].includes(type);
}
`
I would also prefer it if type was passed as an argument since this will make this code less coupled and help with refactoring later.